pkg/controller/master/backup/backup_metadata.go

```diff
@@ -140,7 +143,47 @@ func (h *MetadataHandler) OnBackupTargetChange(_ string, setting *harvesterv1.Se
 		return nil, nil
 	}
 
-	return nil, nil
+	return h.renewBackupTarget(setting)
+}
+
+func (h *MetadataHandler) shouldRefresh(setting *harvesterv1.Setting, refreshIntervalInSeconds int64) bool {
+	if setting.Annotations == nil {
+		return true
+	}
+
+	var err error
+	lastTime := time.Unix(0, 0)
+	currentTime := time.Now()
+	if setting.Annotations[util.AnnotationLastRefreshTime] != "" {
+		lastTime, err = time.Parse(time.RFC3339, setting.Annotations[util.AnnotationLastRefreshTime])
+		if err != nil {
+			logrus.WithError(err).Errorf("failed to parse last refresh time")
+			lastTime = time.Unix(0, 0)
```

Comment:
when return true, `lastTime = time.Unix(0, 0)` seems redundant

Reply:
Removed in https://github.com/harvester/harvester/pull/7684.